### DISCUSSION TITLE
File structure examples

### DISCUSSION THREAD

**Discussion by gdeans** [Lissy93/dashy#1433] [Ideas]
While the examples for things like the widgets is nice, it's incomplete b/c it doesn't show where it needs to be placed within the conf.yml structure.  Some things in the docs are meant to be under specific blocks but it would be nice to have examples of configs that use pages, or have a mixed layout, etc.

**Comment by CrazyWolf13** [comment]
https://dashy.to/docs/configuring

Here it's quite nicely explained what the structure needs to look like.

**Reply by gdeans** [reply]
This doesn't clarify it as much as you think it might.  Most of the component descriptions have snippets that fall under other categories that are assumed you understand.  So I probably missed a single sentence that may have explained it, but Lord knows where it is.  Such as you only need to list widgets: once, then it's a sub list of - type: which may sound normal but it doesn't take someone from knowing nothing about this to being able to create a functional config file.  A fully functioning snippet that could be pasted into the config would be more helpful than a snippet that has to go under something else that isn't discussed.

I've been crawling through the example conf files to help fill in the gaps, but it's still a struggle at times.

**Reply by CrazyWolf13** [reply]
Oh okay.
For me the docs were enough. As I started using dashy I also had basically zero knowledge of linux nor servers nor yaml.

After giving the docs a good read through I was able to create configs by hand without problems, so that's why I suggested that URL.


If you need any further help or specific help, just create an issue with question lable and I'll try to help.

**Reply by CrazyWolf13** [reply]
But I give you the point of less examples.

I will submit a PR to add some more variating examples at the end of the page.

**Reply by CrazyWolf13** [reply]
@gdeans I just noticed at the end of the docs there is a link to more examples which links to this gist:
https://gist.github.com/Lissy93/000f712a5ce98f212817d20bc16bab10

Here are plenty of examples. No need to add more by PR
